diff --git a/src/file_download.h b/src/file_download.h
index 4fa505f..beef8d6 100644
--- a/src/file_download.h
+++ b/src/file_download.h
@@ -98,7 +98,7 @@ class FileDownload
 
 	static int updateProgressHandle(void *p, double dltotal, double dlnow, double ultotal, double ulnow);
 	static size_t writeToStreamHandle(void *buffer, size_t size, size_t nmemb, void *stream);
-	virtual int updateProgress(void *p, double dltotal, double dlnow, double ultotal, double ulnow);
+	int updateProgress(void *p, double dltotal, double dlnow, double ultotal, double ulnow);
 	size_t writeToStream(void *buffer, size_t size, size_t nmemb, void *stream);
 	void initFileToDownload(std::string _url, std::string _file);
 
